mmedit/models/backbones/sr_backbones/tdan_net.py

```diff
@@ -0,0 +1,138 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, constant_init
+from mmcv.ops import DeformConv2dPack, deform_conv2d
+from mmcv.runner import load_checkpoint
+from torch.nn.modules.utils import _pair
+
+from mmedit.models.common import (PixelShufflePack, ResidualBlockNoBN,
+                                  make_layer)
+from mmedit.models.registry import BACKBONES
+from mmedit.utils import get_root_logger
+
+
+class AugmentedDeformConv2dPack(DeformConv2dPack):
+    """Augmented Deformable Convolution Pack.
+
+    Different from DeformConv2dPack, which generates offsets from the
+    preceeding feature, this AugmentedDeformConv2dPack takes another feature to
+    generate the offsets.
+
+    Args:
+        in_channels (int): Same as nn.Conv2d.
```

Comment:
Please refactor docstring of args or copy explain of these args from nn.Conv2d

Reply:
Okay~